environments/key4hep-desy-release/spack.yaml

```
@@ -0,0 +1,21 @@
+spack:
+  config:
+    install_tree:
+      root: $CVMFS_INSTALL_DIR/spackages
+      projections:
+        all: ${PACKAGE}/${VERSION}/${target}-${os}-${COMPILERNAME}${COMPILERVER}-opt/${HASH}
+        build_type=Debug: ${PACKAGE}/${VERSION}/${target}-${os}-${COMPILERNAME}${COMPILERVER}-dbg/${HASH}
```

For key4hep and the lcg-releases a shorter hash is used, keeps path a bit more readable (but maybe you want to the full hash for continuity with previous releases?)
```suggestion
        all: ${PACKAGE}/${VERSION}/${target}-${os}-${COMPILERNAME}${COMPILERVER}-opt/${HASH:5}
        build_type=Debug: ${PACKAGE}/${VERSION}/${target}-${os}-${COMPILERNAME}${COMPILERVER}-dbg/${HASH:5}
```